src/libraries/Common/tests/Extensions/TestingUtils/Microsoft.AspNetCore.Testing/src/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -0,0 +1,50 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It looks like this is eluding our wildcards.  You'll need to add it to ProjectExclude.